src/components/Card.js

```diff
@@ -1,13 +1,34 @@
 export default class Card {
-  constructor({ name, link }, cardSelector, handleCardClick) {
+  constructor(
+    { name, link, owner, _id, likes },
+    cardSelector,
+    handleCardClick,
+    handleDeleteCardConfirmPopup,
+    handleLikeClick,
+    myID
+  ) {
     this.name = name;
     this.link = link;
+    this.ownerId = owner._id;
+    this.myID = myID;
```

Comment:
The variables, which you use unly inside the class should be private

Reply:
I couldn't make this.myID private without getting null errors.

Reply:
You should change `.myID` to `_.myID` everywhere in this file. the same for other variables, which are used only inside this file

Reply:
I changed it to _myID